### PR TITLE
Improve documentation for generating PEP pages.

### PR DESCRIPTION
**Pull request by Mariatta**
Add instruction on how to clone PEP repo.
Also be more specific of where pep2html.py and genpepindex.py should be run from.

### CODE REVIEW

File: docs/source/pep_generation.rst [hunk 1]
```diff
@@ -11,11 +11,20 @@ We are generating the PEP pages by lightly parsing the HTML output from the
...
 The PEP Page Generation process is as follows:
 
-1. Set ``PEP_REPO_PATH`` in ``pydotorg/settings/local.py`` to the location
-   of the `PEP Repository`_.
-2. Run ``pep2html.py``
-3. Run ``genpepindex.py``
-4. After all PEP pages are generated into HTML, run::
+1. Check out PEP Repository, if you have not done so::
+
+      $ hg clone https://hg.python.org/peps/
```
Comment (berkerpeksag) [inline docs/source/pep_generation.rst]:
```
$ git clone https://github.com/python/peps.git
```

File: docs/source/pep_generation.rst [hunk 2]
```diff
@@ -11,11 +11,20 @@ We are generating the PEP pages by lightly parsing the HTML output from the
...
+
+2. From cloned PEP Repository, run::
+
+
```
Comment (berkerpeksag) [inline docs/source/pep_generation.rst]: remove empty line

File: docs/source/pep_generation.rst [hunk 3]
```diff
@@ -11,11 +11,20 @@ We are generating the PEP pages by lightly parsing the HTML output from the
...
+      $ pep2html.py
```
Comment (berkerpeksag) [inline docs/source/pep_generation.rst]: `./pep2html.py` or `python pep2html.py` would be less cryptic.

Comment (berkerpeksag) [inline docs/source/pep_generation.rst]: Alternatively, we can just add `make -j` here.

File: docs/source/pep_generation.rst [hunk 4]
```diff
@@ -11,11 +11,20 @@ We are generating the PEP pages by lightly parsing the HTML output from the
...
-1. Set ``PEP_REPO_PATH`` in ``pydotorg/settings/local.py`` to the location
-   of the `PEP Repository`_.
-2. Run ``pep2html.py``
-3. Run ``genpepindex.py``
-4. After all PEP pages are generated into HTML, run::
+1. Check out PEP Repository, if you have not done so::
+
+      $ hg clone https://hg.python.org/peps/
+
+2. From cloned PEP Repository, run::
```
Comment (berkerpeksag) [inline docs/source/pep_generation.rst]: From _the_ cloned [...]

File: docs/source/pep_generation.rst [hunk 5]
```diff
@@ -11,11 +11,20 @@ We are generating the PEP pages by lightly parsing the HTML output from the
 
 The PEP Page Generation process is as follows:
 
-1. Set ``PEP_REPO_PATH`` in ``pydotorg/settings/local.py`` to the location
-   of the `PEP Repository`_.
-2. Run ``pep2html.py``
-3. Run ``genpepindex.py``
-4. After all PEP pages are generated into HTML, run::
+1. Check out PEP Repository, if you have not done so::
```
Comment (berkerpeksag) [inline docs/source/pep_generation.rst]: Clone the PEP Repository, [...]

File: docs/source/pep_generation.rst [hunk 6]
```diff
@@ -11,11 +11,20 @@ We are generating the PEP pages by lightly parsing the HTML output from the
...
+2. From cloned PEP Repository, run::
+
+
+      $ pep2html.py
+      $ genpepindex.py
+
+3. Set ``PEP_REPO_PATH`` in ``pydotorg/settings/local.py`` to the location
+   of the cloned PEP Repository
+
+4. After all PEP pages are generated into HTML, run in pythondotorg repository::
```
Comment (berkerpeksag) [inline docs/source/pep_generation.rst]: [...] run in your ``pythondotorg`` clone::

**Review by berkerpeksag — CHANGES_REQUESTED** [review]
Could you also update the repo URL in line 57:

```
.. _PEP Repository: https://hg.python.org/peps/
```

**Comment by Mariatta** [comment]
Thanks, @berkerpeksag I made the changes as suggested.

**Review by berkerpeksag — APPROVED** [review]

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**